src/System.ServiceModel.Primitives/tests/Description/OperationBehaviorTest.cs

```diff
@@ -35,4 +37,75 @@ public static void IOperationBehavior_Methods_AreCalled()
         ((ICommunicationObject)proxy).Close();
         ((ICommunicationObject)duplexService).Close();
     }
+
+    [WcfFact]
+    // Validate that we can use XmlSerializerOperationBehavior to modify or add XmlSerializerFormatAttribute on interface operations.
+    public static void XmlSerializerOperationBehavior_BasicUsage()
+    {
+        XmlSerializerOperationBehavior serializerBehavior;
+        BasicHttpBinding binding = new BasicHttpBinding();
+        string baseAddress = "http://localhost:1066/SomeService";
+        ChannelFactory<IXmlTestingType> factory = new ChannelFactory<IXmlTestingType>(binding, new EndpointAddress(baseAddress));
+        ContractDescription cd = factory.Endpoint.Contract;
+        OperationDescriptionCollection collection = cd.Operations;
+
+
+        foreach (OperationDescription description in collection)
+        {
+            // Find the serializer behavior for those operations that have the attribute set via the interface.
+            serializerBehavior = description.Behaviors.Find<XmlSerializerOperationBehavior>();
+            if (serializerBehavior == null)
+            {
+                // This operation was not set with XmlSerializerFormatAttribute
+                // Here we add the attribute programatically using defaults.
+                if (String.Equals(description.Name, "XmlSerializerFormatAttribute_NotSet_One"))
+                {
+                    // Default OperationFormatStyle is "Document"
+                    serializerBehavior = new XmlSerializerOperationBehavior(description);
+                    description.Behaviors.Add(serializerBehavior);
+                }
+                // There is one additional operation not set with XmlSerializerFormatAttribute
+                // Here we add the attribute programatically and further set the OperationFormatStyle to 'Rpc'
+                else
+                {
+                    XmlSerializerFormatAttribute serializerAttribute = new XmlSerializerFormatAttribute();
+                    serializerAttribute.Style = OperationFormatStyle.Rpc;
+
+                    serializerBehavior = new XmlSerializerOperationBehavior(description, serializerAttribute);
+                    description.Behaviors.Add(serializerBehavior);
+                }
+
+            }
+
+            if (String.Equals(description.Name, "XmlSerializerFormatAttribute_Set_StyleSetTo_Rpc") || (String.Equals(description.Name, "XmlSerializerFormatAttribute_NotSet_Two")))
```

Comment:
Use `nameof(IXmlTestingType.XmlSerializerFormatAttribute_Set_StyleSetTo_Rpc)` so that any refactoring won't break the test.